nyc_subway_zoom.star

```diff
@@ -178,6 +177,6 @@ def proto_decode_len(reader):
     return out, reader
 
 def proto_next_byte(reader):
-    out = reader.elem_ords()[0]
```

Comment:
For some reason, `elem_ords()` actually returns the bytes values, while `ord(reader[0])` when `reader` is a string returns 65533 for non-ASCII bytes.